modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="BaselineAdminNetworkPolicy"_{context}"]
+= BaselineAdminNetworkPolicy
+
+`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definitions (CRDs). As an {product-title} administrator, you can use `BaselineAdminNetworkPolicy` to setup and enforce optional baseline network policies that are overridable by users if need be. Rule actions for BANP are `allow` or `deny`.
+
+BANP allows administrators to specify:
+
+* A `subject` that consists of a set of pods that is selected by the policy.
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+[discrete]
+[id="BaselineAdminNetworkPolicy-example"]
+== BaselineAdminNetworkPolicy example
+
+The `BaselineAdminNetworkPolicy` resource is a cluster singleton `object` that can be used as a guardrail policy incase a passed traffic policy does not match any `NetworkPolicies` in the cluster. You must use `default` as the name when creating a `BaselineAdminNetworkPolicy` object.
+
+// Need to confirm with SME if this is correct`BaselineAdminNetworkPolicy` is a `TechnologyPreviewNoUpgrade` feature that can be enabled on test clusters that are not in production. The feature is behind the flag `enable-admin-network-policy`. See "Enabling features using feature gates" in the Additional resources of this section for more information on feature gates and `TechnologyPreviewNoUpgrade` features.
+
+.Example YAML file for BANP
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: priority-50-example <1>
```

Comment:
bad name, there is no priority for BANP
OH WAIT, this is ANP, you need to change this to a BANP..